sys/shell/commands/sc_gnrc_netif.c

```diff
@@ -536,32 +541,32 @@ static void _netif_list(kernel_pid_t iface)
                                    line_thresh);
 #endif
 #endif
-    res = gnrc_netapi_get(iface, NETOPT_SRC_LEN, 0, &u16, sizeof(u16));
+    res = netif_get_opt(iface, NETOPT_SRC_LEN, 0, &u16, sizeof(u16));
     if (res >= 0) {
         printf("Source address length: %" PRIu16 , u16);
         line_thresh++;
     }
     line_thresh = _newline(0U, line_thresh);
 #ifdef MODULE_GNRC_IPV6
     printf("Link type: %s",
-           (gnrc_netapi_get(iface, NETOPT_IS_WIRED, 0, &u16, sizeof(u16)) > 0) ?
```

Comment:
why is this inside `MODULE_GNRC_IPV6` looks like a generic L2 option not deps to GNRC?!

Reply:
I think this option exists only because @OlegHahm was trying to do something about wired vs. not-wired in IPv6, but I can't remember exactly, what it was :-/.

Reply:
Since this PR only renames some functions, I would say this is unrelated

Reply:
agreed, let's just keep this in mind to be improved/changed in another PR